Week1/homework/exercise-1/server.js

```diff
@@ -0,0 +1,48 @@
+// const express = require("express");
+const mysql = require("mysql");
+const connection = mysql.createConnection({
+  host: "localhost",
+  user: "hyfuser",
+  password: "hyfpassword",
+  // database: "meetup",
+  // port : 3307
+  insecureAuth: true,
+});
+
+connection.connect();
+
+const createQuery = [
+  "drop database meetup",
```

Comment:
My server didn't have this database so I got an error on this first query 😝 
not a big problem, but you could write it like
```suggestion
  "DROP DATABASE IF EXISTS meetup",
```